internal/builder/builder.go

```diff
@@ -93,70 +92,62 @@ func constructBuilder(img imgutil.Image, newName string, metadata Metadata) (*Bu
 		return nil, err
 	}
 
-	uid, gid, err := userAndGroupIDs(img)
-	if err != nil {
-		return nil, err
+	bldr := &Builder{
```

Comment:
To simplify the builder construction, I made the basic structure, and then moved all of the various calls to `img.Label` into another helper func (`addImgLabelsToBuildr`)